### PR TITLE
update contribute guide

### PR DESCRIPTION
**Pull request by cldougl**
@bcdunbar please review!

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -56,8 +56,30 @@ exclude: []
...
+git status
+```
+Add, commit, and push the files that you'd like to add to your pr:
+```
+git add file
+git add file
+git commit -m 'message about your changes'
+git push origin your_feature_branch
+```
+Visit the [documentation repo](https://github.com/plotly/documentation) and open a pull request against the `source-design-merge` branch. You can then tag @cldougl and @bcdunbar for a review.
```
Comment (bcdunbar) [inline Contributing.md]: Did we want to include screenshots of this process?

File: Contributing.md [hunk 2]
```diff
@@ -56,8 +56,30 @@ exclude: []
...
+Ready for your changes to be reviewed? Make a pull request against the `source-design-merge` branch! 
+Create a feature branch and use `git status` to list changed files.
+```
+git checkout -b your_feature_branch
+git status
+```
+Add, commit, and push the files that you'd like to add to your pr:
+```
+git add file
+git add file
```
Comment (bcdunbar) [inline Contributing.md]: ⚡️ duplicate

Or do you want them to say file-a and file-b?

Comment (cldougl) [reply Contributing.md]: yeah I want to say file-a and file-b :)

File: Contributing.md (hunk 1 above)
Comment (cldougl) [reply Contributing.md]: @bcdunbar images would be a good idea! I'm going to skip them for this doc but maybe in the individual languages.

**Pull request merged**